jsapp/js/components/activity/activityLogs.query.ts

```diff
@@ -44,6 +45,11 @@ const getActivityLogs = async ({
   );
 };
 
+interface FilterOption {
```

Comment:
There is `LabelValuePair` in `dataInterface.ts` that you can reuse here

Reply:
Thank you!